Sources/SWBMacro/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+
+add_library(SWBMacro
```

Comment:
Is this macro support so that Swift-Build can build macros, or are these macros used in the building of Swift-Build? If the former, this is fine. If the latter, this should be split out into a separate ExternalProject so that cross-compiling will work.

Reply:
SWBMacro is a "macro expansion inside swift-build" thing, not macro-as-compiler-extension. Details about what this is and how it works at https://github.com/swiftlang/swift-build/blob/main/SwiftBuild.docc/Core/macro-evaluation.md

Reply:
Yeah, this is a module that contains the implementation of the "xcconfig language" which is also used for internal rule specifications, etc. No relation to swift macros

Reply:
okay, no changes needed.